Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -6176,6 +6176,7 @@ ThirdPartyIframeRedirectBlockingEnabled:
 ThreadedAnimationResolutionEnabled:
   type: bool
   status: unstable
+  category: animation
```

Comment:
I don't think we've been classify features outside of stable/testable/preview/developer yet.